drivers/telescope/lx200ap_experimental.cpp

```diff
@@ -1693,9 +2213,10 @@ bool LX200AstroPhysicsExperimental::MoveNS(INDI_DIR_NS dir, TelescopeMotionComma
     }
 
     bool rc = LX200Generic::MoveNS(dir, command);
-
+#ifdef no
```

Comment:
Why is this block not in use?

Reply:
There were about four bool variables in use. E.g., motionCommanded was never used. In general I think instead of using a boolean it is better to use the INDI states. I disabled all these variables in favor of a cleaner UnPark() method.